src/components/tabs/types.ts

```diff
@@ -30,6 +30,12 @@ export interface TabsProps {
    */
   initialActiveIndex?: number
 
+  /**
+   * Set to true to enable nested styles. Defaults to false.
+   * Note that allowing nested styles means showAnchorLine will be ignored.
+   */
+  allowNestedStyles?: boolean
```

Comment:
The `allowNestedStyles` prop is set by the consumer. In our case, we only use this in MDX contexts (for now at least).

Reply:
Great question!

I think the subtlety here comes from outer tabs styling, which applies when `allowNestedStyles` is true, to the "outer" tabs. One example would be on install pages, such as [/waypoint/install](https://dev-portal-git-zsstyle-nested-tabs-hashicorp.vercel.app/waypoint/install). [Design spec](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/DevDot?node-id=7738%3A76323) shows the "non-nested" styles, without the extra border and padding on the "outer" tabs.

With this in mind, intent in this PR was to apply the "nested tabs" styles _only_ in MDX.

Reply:
This makes sense! I didn't realize the visual differences were that apparent. I went ahead and added a task for documenting the differences when have time here: https://app.asana.com/0/1202097197789424/1202436586522460/f. 📌